.github/workflows/build_scoped_rector.yaml

```diff
@@ -28,7 +28,7 @@ jobs:
             -
                 name: "Wait 3 minutes before checkout rectorphp/rector-src on create a tag"
                 if: "startsWith(github.ref, 'refs/tags/')"
-                run: sleep 180
+                run: sleep 120
```

Comment:
comment need to be updated to `Wait 2 minutes...`

Reply:
Thanks